heudiconv/convert.py

```diff
@@ -271,16 +270,23 @@ def convert(items, converter, scaninfo_suffix, custom_callable, with_prov,
                                      prefix + scaninfo_suffix)
 
                 if not op.exists(outname) or overwrite:
-                    tmpdir = tempdirs('dcm2niix')
 
+                    # first copy files select files
+                    item_dicoms_dir = tempdirs('DICOMs')
+                    for dcm in item_dicoms:
+                        shutil.copy(
+                            dcm, op.join(item_dicoms_dir, op.basename(dcm)))
```

Comment:
curious: symlinking wouldn't be enough?
I have tested on `///dicoms/dartmouth-phantoms/bids_test3-20161011//bids_anat-T1w` -- seems have worked just fine.  so why not to try to symlink first and then if that fails (file system doesn't support it) - do full copy (I wish there was something like `cp --reflink=auto` in shutils to facilitate fast copy on COW filesystems)